File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main()
{
    int n;
    int a[1000000];
    scanf("%d", &n);
    int min = 1000000;
    int max = -1000000;
    for(int i = 0; i < n; i++)
    {
        scanf("%d", &a[i]);
    }
    for(int j = 0; j < n; j++)
    {
        if(a[j] > max)
        {
            max = a[j];
        }
        if(a[j] < min)
        {
            min = a[j];
        }
    }
    printf("%d %d", min, max);
    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        int[] a = new int[n];
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;

        for (int i = 0; i < n; i++) 
        {
            a[i] = sc.nextInt();
            
            if (a[i] > max) 
            {
                max = a[i];
            }
            if (a[i] < min) 
            {
                min = a[i];
            }
        }

        System.out.printf("%d %d", min, max);
    }
}

